server-monitor

```diff
@@ -4,51 +4,83 @@
             "Address": "ether1.org",
             "Port": 80
         }, {
-            "Name": "ethoFS Website",
+            "Name": "Main ethoFS Website",
             "Address": "ethofs.com",
             "Port": 80
         }, {
-            "Name": "Ether-1 Staking Website",
+            "Name": "Main Ether-1 Staking Website",
             "Address": "staking.ethofs.com",
             "Port": 80
         }, {
-            "Name": "Ether-1 Nodes Website",
+            "Name": "Main Ether-1 Nodes Website",
             "Address": "nodes.ether1.org",
             "Port": 80
 	}, {
-	    "Name": "Ether-1 Wallet Website",
+	    "Name": "Main Ether-1 Wallet Website",
             "Address": "wallet.ether1.org",
             "Port": 80
 	}, {
-            "Name": "Ether-1 Blocks Explorer Website",
+            "Name": "Main Ether-1 Secondary Explorer Website",
             "Address": "blocks.ether1.org",
             "Port": 80
 	}, {
-            "Name": "Ether-1 Explorer Website",
+            "Name": "Main Ether-1 Primary Explorer Website",
```

Comment:
Please change Main to Official & Primary to blockscout

Reply:
Changed Main to Official and Primary to Blockscout